db/schema.rb

```diff
@@ -728,4 +728,34 @@
   add_foreign_key "vita_partner_zip_codes", "vita_partners"
   add_foreign_key "vita_partners", "coalitions"
   add_foreign_key "vita_providers", "provider_scrapes", column: "last_scrape_id"
+
+  create_view "organization_capacities", sql_definition: <<-SQL
+      WITH organization_id_by_vita_partner_id AS (
+           SELECT vita_partners_1.id,
+                  CASE
+                      WHEN (vita_partners_1.parent_organization_id IS NULL) THEN vita_partners_1.id
+                      ELSE vita_partners_1.parent_organization_id
+                  END AS organization_id
+             FROM vita_partners vita_partners_1
+          ), client_ids AS (
```

Comment:
Trivial, but I think it's more readable if the `)` and `client_ids` are on separate lines.

Reply:
in our raw .sql file that's how i have it formatted but i think that scenic turns that into this and i dont have control over it -- i think its better not to touch scenic's generated sql and/or edit the schema file directly!